plugin/evm/vm_validators_test.go

```diff
@@ -99,7 +101,7 @@ func TestValidatorState(t *testing.T) {
 	vm = &VM{}
 	err = vm.Initialize(
 		context.Background(),
-		utils.TestSnowContext(), // this context does not have validators state, making VM to source it from the database
+		utilstest.NewTestSnowContext(t, snowtest.CChainID), // this context does not have validators state, making VM to source it from the database
```

Comment:
We should not use CChainID here and generate a new ID. If we're only using a single chain ID (CChainID) then I think we can just skip taking a ChainID in `NewTestSnowContext` and just define a subnet-evm chain ID and use that.